cmd/skipper/main.go

```diff
@@ -30,11 +30,12 @@ package main
 
 import (
 	"flag"
+	"strings"
+	"time"
+
 	log "github.com/Sirupsen/logrus"
 	"github.com/zalando/skipper"
 	"github.com/zalando/skipper/proxy"
-	"strings"
-	"time"
```

Comment:
why that change in import order? isn't the canonical way to first import stdlib packages and then third party? even [goimports](https://github.com/golang/tools/blob/master/cmd/goimports/goimports.go#L7-L21) does it this way.